scripts/generators/es_template.py

```diff
@@ -279,3 +278,10 @@ def es6_type_fallback(mappings):
                 field_or_multi_field_datatype_defaults(mappings[name])
         if 'properties' in details:
             es6_type_fallback(details['properties'])
+
+
+TYPE_FALLBACKS = {
```

Comment:
Good catch migrating this out of the now removed `schema.oss`.

nit: I prefer defining constants like this at the top of the file since they should be module-wide. Not a strong opinion though, I can see value in keeping the definition close to the single use of it here.

Reply:
That's a fair point, I'll move